src/TrueOrFalse.Frontend.Web/Program.cs

```diff
@@ -43,12 +45,18 @@
 
 Settings.Initialize(builder.Configuration);
 
-builder.Services.AddSession(options =>
+var useRedis = builder.Configuration.GetValue<bool>("Redis:Use");
+if (useRedis)
+    Redis.Configuration(builder.Services);
+else
 {
-    options.Cookie.IsEssential = true;
-    options.Cookie.HttpOnly = true;
-    options.IdleTimeout = TimeSpan.FromMinutes(Settings.SessionStateTimeoutInMin);
-});
+    builder.Services.AddSession(options =>
+    {
+        options.Cookie.IsEssential = true;
```

Comment:
is not related to session storage, should be same for redis and in memory session handling